src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.test.ts

```diff
@@ -237,6 +238,8 @@ jest.mock('../../../../state/transactions/hooks', () => {
 
 describe('parseLocalActivity', () => {
   it('returns swap activity fields with known tokens, exact input', () => {
+    const { formatNumber } = renderHook(() => useFormatter()).result.current
```

Comment:
could you define this one globally or beforeEach to avoid the repetition in this file?

Reply:
the hook has to be rendered inside of the test